osu.Game.Rulesets.Osu/Skinning/Default/SliderBall.cs

```diff
@@ -103,14 +103,22 @@ private set
                 tracking = value;
 
                 if (InputTracksVisualSize)
-                    followCircle.ScaleTo(tracking ? 2.4f : 1f, 300, Easing.OutQuint);
+                {
+                    if (tracking)
+                        followCircle.ScaleTo(2f, 200, Easing.OutQuint);
```

Comment:
this makes skins track (as per stable's weirndess) an actually incorrect visual area which was intended to be improved in lazer. you're undoing this improvement.

Reply:
Ah, makes sense. Reverted to 2.4x size.